pkg/neutronapi/dbsync.go

```diff
@@ -1,28 +1,34 @@
 package neutronapi
 
 import (
+	common "github.com/openstack-k8s-operators/lib-common/pkg/common"
 	neutronv1beta1 "github.com/openstack-k8s-operators/neutron-operator/api/v1beta1"
-	common "github.com/openstack-k8s-operators/neutron-operator/pkg/common"
 	batchv1 "k8s.io/api/batch/v1"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/runtime"
-	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 )
 
 // DbSyncJob func
-func DbSyncJob(cr *neutronv1beta1.NeutronAPI, scheme *runtime.Scheme) *batchv1.Job {
+func DbSyncJob(
+	cr *neutronv1beta1.NeutronAPI,
+	labels map[string]string,
+) *batchv1.Job {
 
 	runAsUser := int64(0)
 	initVolumeMounts := GetInitVolumeMounts()
 	volumeMounts := GetAPIVolumeMounts()
 	volumes := GetAPIVolumes(cr.Name)
+	cmLabels := common.GetLabels(cr, common.GetGroupLabel(ServiceName), map[string]string{})
+
+	envVars := map[string]common.EnvSetter{}
+	envVars["KOLLA_CONFIG_FILE"] = common.EnvValue("/var/lib/config-data/merged/db-sync-config.json")
```

Comment:
Optional: could make the `/var/lib/config-data/merged/db-sync-config.json` a const in your `const.go` in the package